### PR TITLE
Add integration test for nomdex

### LINKED ISSUE #2414: Nomdex needs integration test.

### PR DESCRIPTION
**Pull request by willhite**
Fixes #2414

### CODE REVIEW

File: samples/go/nomdex/nomdex_test.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    "github.com/attic-labs/noms/go/util/clienttest"
+    "github.com/attic-labs/testify/suite"
+)
+
+func TestCSVImporter(t *testing.T) {
+    suite.Run(t, &testSuite{})
+}
+
+type TestObj struct {
+    Key  int
```
Comment (arv) [inline samples/go/nomdex/nomdex_test.go]: go format

File: samples/go/nomdex/nomdex_test.go [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+func TestNomdex(t *testing.T) {
+    suite.Run(t, &testSuite{})
+}
+
+func makeTestDb(s *testSuite, dsId string) datas.Database {
+    db := datas.NewDatabase(chunks.NewLevelDBStore(s.LdbDir, "", 1, false))
+    fmt.Println("ldbDir:", s.LdbDir)
+    l1 := []TestObj{
+        TestObj{1, "will", "smith", "m", 40},
```
Comment (arv) [inline samples/go/nomdex/nomdex_test.go]: @mikegray just fixed a bunch of these. The struct name should be omitted here.
go report finds these... I'm not sure how to run that locally.

File: samples/go/nomdex/nomdex_test.go [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    clienttest.ClientTestSuite
+}
+
+func TestNomdex(t *testing.T) {
+    suite.Run(t, &testSuite{})
+}
+
+func makeTestDb(s *testSuite, dsId string) datas.Database {
+    db := datas.NewDatabase(chunks.NewLevelDBStore(s.LdbDir, "", 1, false))
+    fmt.Println("ldbDir:", s.LdbDir)
```
Comment (arv) [inline samples/go/nomdex/nomdex_test.go]: is this print needed?

Comment (willhite) [inline samples/go/nomdex/nomdex_test.go]: Nope. It's gone now.

File: samples/go/nomdex/nomdex_test.go (hunk 2 above)
Comment (willhite) [inline samples/go/nomdex/nomdex_test.go]: ok, 'gofmt -s' was the magic command.

**Comment by arv** [comment]
LGTM

**Pull request merged**